packages/js/src/first-time-configuration/tailwind-components/steps/indexation/configuration-indexation.js

```diff
@@ -5,12 +5,6 @@ import { Transition } from "@headlessui/react";
 import Indexation from "./indexation";
 import Alert from "../../base/alert";
 
-const preIndexingActions = {};
-const indexingActions = {};
-
-window.yoast = window.yoast || {};
-window.yoast.indexing = window.yoast.indexing || {};
-
```

Comment:
You removed some defensive coding here. Are you certain we'll never call `window.yoast.indexing.preIndexingActions` without the object being there?